configs/test/bot/setup/android.bash

```diff
@@ -98,7 +110,10 @@ rm -rf clusterfuzz
 $GSUTIL_PATH/gsutil cp gs://$DEPLOYMENT_BUCKET/linux.zip clusterfuzz-source.zip
 unzip -q clusterfuzz-source.zip
 
-echo "Running ClusterFuzz."
-OS_OVERRIDE="ANDROID" ANDROID_SERIAL="$ANDROID_SERIAL" PATH="$PATH" NFS_ROOT="$NFS_ROOT" GOOGLE_APPLICATION_CREDENTIALS="$GOOGLE_APPLICATION_CREDENTIALS" ROOT_DIR="$ROOT_DIR" PYTHONPATH="$PYTHONPATH" GSUTIL_PATH="$GSUTIL_PATH" python $ROOT_DIR/src/python/bot/startup/run.py &
-
-echo "Success!"
+if [ -z "$ANDROID_SERIAL" ]; then
+  for serial in `adb devices -l | awk -F' ' '{ print $3 }' | grep usb:`; do
```

Comment:
since this is open-source version, i think it should rely on actual serial than the usb version.

Reply:
I suppose that would be more resilient to device reconnects, but I don't like the idea of maintaining two different versions of this. It's probably not going to be much of a burden so making the change for now, and we can revisit it later if it causes any trouble down the road.